docs/examples/offloading-multiple-pods.md

```diff
@@ -0,0 +1,251 @@
+```
+
+At this point, you should have three clusters with Liqo installed on them.
+The setup script named them **origin**, **destination1** and **destination2**, and respectively configured the following cluster labels:
+
+* *origin*: `topology.liqo.io/type=origin`
+* *destination1*: `topology.liqo.io/type=destination`
+* *destination2*: `topology.liqo.io/type=destination`
+
+You can check that the clusters are correctly labeled through:
+
+```bash
+liqoctl status
+liqoctl status --kubeconfig "$KUBECONFIG_DESTINATION1"
+liqoctl status --kubeconfig "$KUBECONFIG_DESTINATION1"
+```
+
+## Peer the clusters
+
+Once Liqo is installed in your clusters, you can establish new *peerings*.
+In this example, you will use the [in-band peering approach](FeaturesPeeringInBandControlPlane).
+
+To establish the peerings from *origin* to *destination* clusters run:
+
+```bash
+liqoctl peer in-band --remote-kubeconfig "$KUBECONFIG_DESTINATION1"
+liqoctl peer in-band --remote-kubeconfig "$KUBECONFIG_DESTINATION2"
+```
+
+When the above commands return successfully, you can check the peering status by running:
+
+```bash
+kubectl get foreignclusters
+```
+
+The output should look like the following, indicating that an outgoing peering is currently active towards both the *destination1* and the *destination2* clusters, as well as the cross-cluster network tunnels have been established:
+
+```text
+NAME           TYPE     OUTGOING PEERING   INCOMING PEERING   NETWORKING    AUTHENTICATION   AGE
+destination1   InBand   Established        None               Established   Established      41s
+destination2   InBand   Established        None               Established   Established      7s
+```
+
+Additionally, you should have two new virtual nodes in the *origin* cluster, characterized by the install-time provided labels:
+
+```bash
+kubectl get node --selector=liqo.io/type=virtual-node --show-labels
+```
+
+```text
+NAME            STATUS   ROLES   AGE     VERSION   LABELS
+NAME                STATUS   ROLES   AGE   VERSION   LABELS
+liqo-destination1   Ready    agent   57s   v1.25.0   liqo.io/remote-cluster-id=df0baf71-c428-4d2e-8c97-3559e205d34a,liqo.io/type=virtual-node,topology.liqo.io/type=destination
+liqo-destination2   Ready    agent   43s   v1.25.0   liqo.io/remote-cluster-id=3c4604aa-b9b4-4442-be6d-8ffc713139fc,liqo.io/type=virtual-node,topology.liqo.io/type=destination
+```
+
+```{admonition} Note
+Some of the default labels were omitted for the sake of clarity.
+```
+
+## Tune namespace offloading
+
+Now, let's suppose you want to deploy an application that needs to be scheduled on every *destination* cluster, but not in the *origin* one.
+
+```bash
+kubectl create namespace liqo-demo
+```
+
+Then, enable Liqo offloading for that namespace:
+
+```bash
+liqoctl offload namespace liqo-demo \
+  --namespace-mapping-strategy EnforceSameName \
+  --pod-offloading-strategy Remote \
+  --selector 'topology.liqo.io/type=destination'
```

Comment:
This is useless: you are selecting all clusters

Reply:
I would keep it anyway as an example, e.g., to show how to deal with the case in which you have two types of edge clusters (edge-italy, edge-france).

Reply:
I know is useless , but I did it to show how to manage a case in which not all remote clusters have to be considered for the offloading. I could:
1.  add a **note** to explain that and keeps the command as it is
2.  add a **note** to explain that is possible to add the flag --selector to specify the destination clusters and remove the --selector flag from the command.

Reply:
In my opinion, the second is better. You can say something like: "You can select a subset of remote clusters by using the `--selector` flag (e.g., selecting all clusters in the region `edge-italy`. For further details, try https://docs.liqo.io/en/v0.6.1/examples/offloading-with-policies.html#tune-namespace-offloading"